packages/rum-core/src/common/config-service.js

```diff
@@ -91,7 +91,8 @@ class Config {
       monitorLongtasks: true,
       apiVersion: 2,
       context: {},
-      session: false
+      session: false,
+      apmServerRequest: null
```

Comment:
we can just keep it as `onRequest` to keep it simpler? Thoughts?

Reply:
I thought about `onRequest` but I think that's too generic and might be confused for all http request (regards of the apm server)

Reply:
Hmmm. `apmRequest`? I was thinking about users building their own backend, but dont expect it to support our format though. Anyways I am fine with as it is or `apmRequest` up to you.